cockpit/src/subscriptions-register.jsx

```diff
@@ -32,7 +32,7 @@ import * as Insights from './insights.jsx';
  *   - onChange  callback to signal when the data has changed
  *   - properties as in defaultRegisterDialogSettings()
  */
-class PatternDialogBody extends React.Component {
+class SubscriptionsRegister extends React.Component {
```

Comment:
Can you please rename `SubscriptionsRegister` to something like `SubscriptionRegisterDialog`, because `SubscriptionsRegister` is not too much descriptive.

Reply:
Fixed